docs/faq/faq.md

```diff
@@ -0,0 +1,271 @@
+# Create® 3 Educational Robot FAQs
+
+## About the Create® 3 educational robot
+
+### What is the Create® 3 educational robot?
+The Create® 3 robot is a programmable robot based on the iRobot® Roomba® i3 vacuum robot.
+Users can program this robot using ROS 2 or Python.
+There have been three versions of the Create® robot series:
+
+- The original Create® robot, released in 2007. This robot was based on the Roomba® Discovery series robots. It is all-white.
+- The Create® 2 robot, released in 2014. This robot is based on the Roomba® 650 robot and includes a green faceplate marked with safe places to drill into the faceplate, once removed from the robot, to mount additional hardware.
+- The Create® 3 robot, released in 2022. This robot is based on the Roomba® i3 robot and includes a translucent black faceplate and custom payload bay with holes for mounting.
+
+----
+
+### What age(s) is the Create® 3 educational robot designed for?
+The Create® 3 educational robot is a STEM resource for educators, students, and developers.
+It is an advanced platform intended for college and professional learners.
```

Comment:
In the FAQ below we also mention "upper high school", should we include it also here?

Reply:
resolved in upcoming commit by removing "upper high school"